test/e2e/alert.test.js

```diff
@@ -60,10 +61,9 @@ describe('vl-alert', async () => {
         await assert.eventually.isTrue(alert.isLarge());
     });
 
-    after((done) => { 
-        if (driver) {
-            driver.quit();
-        }
-        done();
+    it('Als ik een alert met actions defineer, kan ik hiervan de tekst opvragen', async() => {
+        await assert.eventually.equal(vlAlertPage.getTekstVanButtonVanCtaAlert(), 'Fout melden');
```

Comment:
zouden we hier niet beter chainen en vlAlertPage.get...Alert().getButton().getText() doen?

Reply:
Ik zou ook een functie `getButtonVanCtaAlert` voorzien die een VlButton returnt zodat in de test al de functies voorzien op het page object van VlButton beschikbaar zijn.